### ISSUE TITLE
PyAMG.jl wrapper

### ISSUE THREAD

**Issue by ChrisRackauckas** [SciML/LinearSolve.jl#372]

**Comment by AJ0070** [comment]
@ChrisRackauckas

I have implemented the wrapper (`PyAMGJL`, `LinearSolvePyAMGExt.jl`, Project.toml weakdep, and docs), but [`cortner/PyAMG.jl`](https://github.com/cortner/PyAMG.jl) is not in the Julia General registry (only the old METADATA.jl era), so `pkg> add PyAMG` won't work and the weak dependency won't resolve.

Is there any way to have @cortner register it via [Registrator.jl](https://github.com/JuliaRegistries/Registrator.jl)?

**Comment by ChrisRackauckas** [comment]
it doesn't have much code and it doesn't look maintained, so we can just add it as like a LinearSolvePyAMG.jl subfolder.

**Comment by cortner** [comment]
It is not maintained - feel free to take it over or do as Chris suggests.

**Comment by AJ0070** [comment]
yeah I have already started working on the approch chris suggested
